packages/embedded/package.json

```diff
@@ -1,13 +1,13 @@
 {
-  "name": "@cerbos/lite",
+  "name": "@cerbos/embedded",
   "version": "0.5.0-beta",
```

Comment:
drop `beta`?

Reply:
Yep, I will bump the version to 0.6.0 in a subsequent release PR (I also want to release patch versions of the other packages).